### PR TITLE
Use Schema Loader jar to load schema in ScalarDB GraphQL Sample

### PR DESCRIPTION
**Pull request by brfrn169**
Please take a look!

### CODE REVIEW

File: scalardb-graphql-sample/README.md [hunk 1]
```diff
@@ -35,11 +35,11 @@ You need to wait for the Cassandra to start up. Checking the logs with the `dock
...
 In this step, you will load the database schema for the sample app (`emoney-schema.json`) to the Cassandra database.
+Please download the schema tool `scalardb-schema-loader-<version>.jar` that can be found in [releases](https://github.com/scalar-labs/scalardb/releases) of ScalarDB.
+You can load the schema to the database by the following command:
 
-Since the `schema-loader` service is defined in `docker-compose.yml` as a one-shot service (with `profiles`), you can load the schema to the database by the following command:
-
-```console
-docker-compose run --rm schema-loader
+```shell
+$ java -jar scalardb-schema-loader-<version>.jar --config database.properties --schema-file emoney-schema.json --coordinator
```
Comment (brfrn169) [inline scalardb-graphql-sample/README.md:42]: Previously, we loaded the schema using the docker-compose command. However, I've modified the process to download the schema loader jar and load the schema using that instead.

**Review by komamitsu — APPROVED** [review]
LGTM! 👍

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**